fe/fe-core/pom.xml

```diff
@@ -227,6 +227,22 @@ under the License.
             <classifier>jar-with-dependencies</classifier>
         </dependency>
 
+        <!-- https://mvnrepository.com/artifact/com.baidu/jprotobuf-precompile-plugin -->
+        <dependency>
+            <groupId>com.baidu</groupId>
+            <artifactId>jprotobuf-precompile-plugin</artifactId>
```

Comment:
is it necessary to mark this dependency as `compile` dependency?

Reply:
Sounds great. This plugin is only used during compilation, so it can be changed to `provided`.